R/median_normalisation.R

```diff
@@ -1,29 +1,36 @@
 #' Median normalisation
 #'
-#' Median normalise intensity values for all samples of mass spectrometry data. The intensity values are ideally log2 transformed. The output is a new column called normalised_intensity_log2.
+#' Normalises the intensities in the way this is implemented in Spectronaut. The normalised intensity is the original intensity minus the run median plus the global median.
 #'
-#' @param data A data frame containing at least sample names and intensity values.
-#' @param file_column_name The name of the column containing the sample names.
-#' @param intensity_column_name The name of the column containing the intensity values.
-#' @param na.rm A logical indicating wther missing values should be removed.
+#' @param data A data frame containing at least sample names, grouping variables and intensity values.
+#' @param sample The name of the column containing the sample names.
+#' @param grouping The name of the column containing the grouping variable - this can be peptides, precursors or proteins.
+#' @param log2intensity The name of the column containing the log2 transformed intensity values to be normalised.
+#' @param na.rm Logical indicating whether missing values should be removed. Default is TRUE.
 #'
-#' @return A data frame that contains the input data and an additional column with normalised intensity values.
+#' @return A new column in the original dataframe called \code{normalised_intensity_log2} containing the normalised intensity values.
```

Comment:
I would still say the return is a data frame, since this is a case. Otherwise it sounds like you could just run the function like this: function(data, y, z) and not like this data2 <- function(data, y, z)
It does not modify the original data frame, it creates a new one that is the original with another column.